OG_Format.adoc

```diff
@@ -268,8 +268,8 @@ OG1.0 requirements cover positioning variables and geolocation of any scientific
 * data type: double
 * dimension: N_MEASUREMENTS |
 
-* long_name = “longitude of each measurement and GPS location”;
-* standard_name = “longitude”;
+* long_name = “Longitude east”;
```

Comment:
It is often used only "Longitude" as `long_name`. Many applications automatically add units right after, so in that case it would result in `Longitude east [degrees_east]`. But a valid choice if you want that way.